floto/settings.py

```diff
@@ -87,12 +92,24 @@
 # Database
 # https://docs.djangoproject.com/en/4.1/ref/settings/#databases
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+if os.getenv("DB_ENGINE"):
```

Comment:
thank you! I didn't commit this yet, my apologies

Reply:
No worries :)